test/beast/core/make_strand.cpp

```diff
@@ -28,7 +28,7 @@ class make_strand_test : public beast::unit_test::suite
         net::make_strand(ioc.get_executor());
         net::make_strand(net::make_strand(ioc));
 
-        net::executor ex(ioc.get_executor());
+        net::any_io_executor ex(ioc.get_executor());
         net::make_strand(ex);
 
```

Comment:
make_strand.cpp is unnecessary, it used to test Beast's `make_strand` function which existed before it was added to Asio. No Beast release ever shipped a `make_strand` function though.

Reply:
would you like it removed?